lib/stats.rb

```diff
@@ -67,6 +67,10 @@ def uptime
       (Time.now - Time.parse(@wstats['started_at'])).to_i
     end
 
+    def requests_count
+      @wstats.dig('last_status', 'requests_count') || @wstats['requests_count'] || 0
```

Comment:
:thinking: I'm wondering if it would be better to not display anything if stats do not include the requests_count key.
`|| 0` here makes the if condition in https://github.com/ylecuyer/puma-status/pull/3/files#diff-58e9cdf303f25150f127be9260612110R65 meaningless

Reply:
I didn't check the code but definitely if the number is not present it should not be shown